zsh/zshrc.symlink

```diff
@@ -12,30 +9,31 @@ then
   source ~/.localrc
 fi
 
-# all of our zsh files
-typeset -U config_files
-config_files=($ZSH/**/*.zsh)
+# all of our non-path/env/profile/login/logout zsh files
+typeset -U fpath_files interactive_files completion_files
+fpath_files=($DOTFILES/**/*fpath.zsh)
+interactive_files=($DOTFILES/**/*.zsh~**/*completion.zsh~**/*path.zsh~**/*env.zsh~**/*alias.zsh~**/*aliases.zsh~**/*profile.zsh~**/*login.zsh~**/*logout.zsh)
```

Comment:
I had to change `~**/*aliases.zsh` to `#**/*aliases.zsh`, otherwise it didn't pick up aliases defined in `zsh/aliases.zsh`, but I don't know what I'm doing here...